sdcm/utils/full_storage_utils.py

```diff
@@ -0,0 +1,117 @@
+import logging
+
+LOGGER = logging.getLogger(__name__)
+
+class DiskUtils:
+    @staticmethod
+    def get_disk_info(node):
+        result = node.remoter.run("df -h -BG --output=size,used,avail,pcent /var/lib/scylla | sed 1d | sed 's/G//g' | sed 's/%//'")
+        size, used, avail, pcent = result.stdout.strip().split()
+        return {
+            'total': int(size),
+            'used': int(used),
+            'available': int(avail),
+            'used_percent': int(pcent)
+        }
+
+    @staticmethod
+    def get_max_disk_usage(nodes):
+        max_usage = 0
+        max_used = 0
+        for node in nodes:
+            info = DiskUtils.get_disk_info(node)
+            max_usage = max(max_usage, info["used_percent"])
+            max_used = max(max_used, info["used"])
+        return max_usage, max_used
+
+    @staticmethod
+    def calculate_target_used_size(nodes, target_percent):
```

Comment:
ok, added comments and renamed the function. Regarding mixed cluster, @cezarmoise is working on those cases, his PR should be handling them. (It will be ready for review once this gets merged).